src/main/java/com/wellsmith/sapien/persistence/json/JsonEventSerializationSchema.java

```
@@ -0,0 +1,32 @@
+package com.wellsmith.sapien.persistence.json;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.afterburner.AfterburnerModule;
+import com.wellsmith.phoenix.commonjava.event.Event;
+import java.io.IOException;
+import org.apache.flink.api.common.serialization.SerializationSchema;
+import org.apache.flink.api.common.serialization.SimpleStringSchema;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JsonEventSerializationSchema<T extends Event> implements SerializationSchema<T> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(JsonEventSerializationSchema.class);
+  private static final SimpleStringSchema serializer = new SimpleStringSchema();
+  private static final ObjectMapper mapper = new ObjectMapper();
+
+  static {
+    mapper.registerModule(new AfterburnerModule());
+  }
+
+  @Override
+  public byte[] serialize(T event) {
+
+    try {
+      return serializer.serialize(mapper.writeValueAsString(event));
+    } catch (IOException ioe) {
+      LOG.error("Error serializing Event", ioe);
```

same thing here about exceptions. i'd recommend failing fast by rethrowing the exception although i understand we want to do something more sophisticated later.

---

I don't want the entire workflow to fail just because one object failed to serialize.  Plus, the method I'm overriding doesn't throw an Exception (probably for that very reason), so I'd have to throw a RuntimeException...